src/actions/reported.actions.js

```diff
@@ -51,9 +51,28 @@ const requestReview = () => {
   };
 };
 
+const reportArticle = (articleid, body) => {
+  return async dispatch => {
+    dispatch(contentLoading());
+    try {
+      const { data } = await http.post(`/article/report/${articleid}`, body);
+      toast.info(data.message, {
+        type: toast.TYPE.INFO,
+        closeButton: false,
```

Comment:
Nice work. However, you can add a cancel icon to this toaster by setting `closeButton: true`